pkg/audit/verifier.go

```diff
@@ -416,6 +421,11 @@ func (verifier *Verifier) Reverify(ctx context.Context, stripe *Stripe) (report
 			err = errs.Combine(err, result.err)
 		}
 	}
+	//remove failed audit pieces from the pointer so as to only penalize once for failed audits
+	err = verifier.RemoveFailedPieces(ctx, stripe, report.Fails)
```

Comment:
This is not really correct. Reverify is not executed on the selected stripe, but on the stripe in the pending audit of reverified nodes. So, if a node failed reverification on the stripe in the pending audit, this does not mean that it is a bad node for the selected stripe too (which it was not really audited on).

Reply:
will fix